Week 1/Weekly Practice Problems (Week 1)/1929. Concatenation of Array/dominic.cpp

```diff
@@ -0,0 +1,8 @@
+class Solution {
+public:
+    vector<int> getConcatenation(vector<int>& nums) {
+        int n = nums.size();
+        for (int i = 0; i < n; i++) nums.push_back(nums[i]);
```

Comment:
oh, lol. I think there's a vector::insert function as well

Reply:
Oh huh, I don't think I've ever used insert like this. I'll make an implementation of it quickly.